$NetBSD: patch-af,v 1.5 2014/10/12 14:58:27 wiz Exp $

Make it clear that this is GNU make and not BSD make.

--- doc/make.info.orig	2014-10-05 16:25:47.000000000 +0000
+++ doc/make.info
@@ -24,8 +24,9 @@ Manual', for GNU 'make' version 4.1.
      developing GNU and promoting software freedom."
 INFO-DIR-SECTION Software development
 START-INFO-DIR-ENTRY
-* Make: (make).            Remake files automatically.
+* GNU Make: (make).            Remake files automatically.
 END-INFO-DIR-ENTRY
+ 
 
 
 Indirect:
